project-1/requirements.txt

```diff
@@ -0,0 +1,5 @@
+#Selenium version 4.14.0
```

Comment:
This is not the proper way to format a requirements.txt file.
Run `pip freeze > requirements.txt` to generate a requirements file. It would add all packages in the environment and not just the ones you used in your project, it's just so you can see the proper way to format the file.